src/core_codemods/enable_jinja2_autoescape.py

```diff
@@ -0,0 +1,34 @@
+from codemodder.codemods.base_codemod import ReviewGuidance
+from codemodder.codemods.api import SemgrepCodemod
+
+
+class EnableJinja2Autoescape(SemgrepCodemod):
+    NAME = "enable-jinja2-autoescape"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_AFTER_CURSORY_REVIEW
+    SUMMARY = "Enable jinja2 autoescape"
+    DESCRIPTION = "Makes the `autoescape` parameter to jinja2.Environment be `True`."
+
+    @classmethod
+    def rule(cls):
+        return """
+            rules:
+                - patterns:
+                  - pattern-either:
+                    - pattern: |
+                        jinja2.Environment()
+                    - pattern: |
+                        jinja2.Environment(...)
+                    - pattern: |
+                        jinja2.Environment(..., autoescape=False, ...)
+                  - pattern-not: |
```

Comment:
Wouldn't this `pattern-not` by itself be sufficient instead of enumerating the other cases above?

Reply:
oh hmm good point let me check if tests pass with that